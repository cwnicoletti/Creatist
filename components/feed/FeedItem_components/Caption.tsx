import React from "react";
import { StyleSheet, Text, View } from "react-native";

const Caption = (props) => {
  return (
    <View style={{ ...styles.container, ...props.captionContainer }}>
      <Text style={{ ...styles.caption, ...props.captionStyle }}>
        {props.caption}
      </Text>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    justifyContent: "center",
  },

  caption: {
    textAlign: "center",
    marginVertical: 10,
    marginHorizontal: "10%",
    fontSize: 13,
  },
});

export default Caption;
