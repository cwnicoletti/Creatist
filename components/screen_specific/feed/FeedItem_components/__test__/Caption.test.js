import React from "react";
import renderer from "react-test-renderer";
import Caption from "../Caption";

test("matches previous snapshot and renders correctly", () => {
  const tree = renderer.create(<Caption />).toJSON();
  expect(tree).toMatchSnapshot();
});
