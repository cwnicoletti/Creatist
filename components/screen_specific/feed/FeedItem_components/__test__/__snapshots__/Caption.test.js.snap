// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches previous snapshot and renders correctly 1`] = `
<View
  style={
    Object {
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 13,
        "marginHorizontal": "10%",
        "marginVertical": 10,
        "textAlign": "center",
      }
    }
  />
</View>
`;
